packages/user-interface/src/hooks/useNotification.tsx

```diff
@@ -0,0 +1,8 @@
+import { useContext } from "react";
+import NotificationContext from "../contexts/NotificationContext";
+
+const useNotification = () => {
```

Comment:
Twijfel altijd een beetje over het aanroepen van context met een hook. Is wel handig als je kan checken of de context beschikbaar is. Maar het is ook een beetje overhead. Heb jij hier nog aanvullingen op waarom wel of niet?

Reply:
Heb de hook weggehaald